composer.json

```diff
@@ -13,14 +13,15 @@
     ],
     "require": {
         "chameleon-system/chameleon-shop": "dev-master",
+        "chameleon-system/chameleon-base": "dev-master",
```

Comment:
This require shouldn't be necessary.

Reply:
oh no.. i added them to ensure the installation of the upgrade branch. Instead of removing them I switched them to master, which is not very smart of me. I'll remove them